src/transformers/tokenization_utils_base.py

```diff
@@ -2417,6 +2424,9 @@ def _from_pretrained(
                 "Special tokens have been added in the vocabulary, make sure the associated word embeddings are"
                 " fine-tuned or trained."
             )
+        if tokenizer.vocab_size > 100000:
+            # Try to catch mistral tokenizers.
+            pass # TODO
```

Comment:
@patrickvonplaten that's were I think it would make sense to post-process the `regex`. I just want to avoid downloading too many files for everyone (especially the config) so any common denomitors are welcome!